Formula/h/hyperkit.rb

```diff
@@ -4,12 +4,10 @@ class Hyperkit < Formula
   url "https://github.com/moby/hyperkit/archive/refs/tags/v0.20210107.tar.gz"
   sha256 "095f5f5ef550d7cad10e4d13e9c9ce8b58cc319d654a6d837d8d87ee70537835"
   license "BSD-2-Clause"
+  revision 1
 
   bottle do
-    sha256 cellar: :any_skip_relocation, ventura:  "3b67078315551718bc3c752b943b933713ddb69058f3cb72a0f65faa6e9295ab"
-    sha256 cellar: :any_skip_relocation, monterey: "da3b0d0374a85af5c649c86fb7796c1eecae468f5783bbb994a96d807e60712a"
-    sha256 cellar: :any_skip_relocation, big_sur:  "f96e7270e9e853ce33f2195136b11338a5cf4d612ee50f3dd51b5c8506b4efcb"
-    sha256 cellar: :any_skip_relocation, catalina: "cd58afe172473278d3ed9404e9d25e10bee487fb4e27cd6de39c950a0ccaca87"
```

Comment:
need to followup on this

Reply:
https://github.com/Homebrew/homebrew-core/actions/runs/9394971412

Reply:
Hyperkit really looks like abandonware now: 
- https://github.com/moby/hyperkit/commits/master/
- https://github.com/moby/hyperkit/issues
- https://github.com/moby/hyperkit/pulls 

Not much could probably be done for it, until developers/maintainers step in.

Reply:
Apologize for tagging you @avsm, but since you were hyperkit developer, you might be able to help reviving it.

Reply:
yeah, there are no new commits for the past two years.

Reply:
It's not so much abandonware, but stable. There's no need to keep it up-to-date with the very latest OCaml, and I doubt there are many users accessing it via Homebrew. I'm happy to it to be removed from Homebrew to ensure there's no blocker on upstream OCaml updates. Copying in @djs55 as another maintainer of Hyperkit to confirm.

Reply:
I would describe it as stable -- it's still shipped in Docker Desktop for Intel Macs. Removing hyperkit from home-brew is fine.